tools/dataProtector/dataProtectorCore/revokeAllAccess.md

```diff
@@ -72,12 +72,12 @@ const revokeAllAccessResult = await dataProtectorCore.revokeAllAccess({
 
 ### authorizedUser
 
-`AddressOrENS | 'any' | undefined`
-
-_default_: `any`
+`AddressOrENS | undefined`
 
 The user address to be removed from the authorization list for the specified
-`protectedData`.
+`protectedData`. If no address is specified, it will return the granted access
+that authorized any user address (equivalent to
+`0x0000000000000000000000000000000000000000`).
```

Comment:
Same here, better to speak about "it will revoke access to..." instead of "it will return...". (That was correct for read functions but less here)

Also I would remove the mention to the Zero address as I find it confusing.